quiz-app/src/containers/quiz.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Answers from '../components/answers';
+import Questions from '../components/questions';
+
+const Quiz = ({results}) => {
+    let i = 0;
+    const counter = (i) => i++;
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
```

Comment:
We can also add the logic to check if answer is correct here as well as dispatch the response